### DISCUSSION TITLE
Rewrite help for my vercel.json

### DISCUSSION THREAD

**Discussion by waferbaby** [vercel/community#2937] [Help]
### Summary

Hey friends! Just switched over to Vercel, loving it so far. Trying to get rewrites working for my (static!) API - so from:

`https://usesthis.com/api/interviews/?page=2`

to

`https://usesthis.com/api/interviews/page_2.json`

Here's my config currently, which I'm clearly doing something wrong with, because it doesn't work :)

```
{
  "rewrites": [
    {
      "source": "/api/:path*",
      "has": [
        {
          "type": "query",
          "key": "page",
          "value": "(?<page>.*)"
        }
      ],
      "destination": "/api/:path*/page_:page.json"
    }
  ]
}
```

Would love a hand fixing this. Thank you!

### Steps to Reproduce

Try out the above config file.

**Comment by star-starry-sea** [comment]
I have the problem too! This is my discussion: https://github.com/orgs/vercel/discussions/3588
